index.ts

```
@@ -0,0 +1,134 @@
+ }
+});
+
+// Route to get all published posts
+app.get('/posts', async (req: Request, res: Response) => {
+ const { page = 1, limit = 10 } = req.query;
+
+ const pageNumber = parseInt(page as string, 10);
+ const limitNumber = parseInt(limit as string, 10);
+
+ if (
+  isNaN(pageNumber) ||
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)